search/lib/api/provider.js

```diff
@@ -16,31 +16,19 @@ async function getProvider (request, response) {
     const isProvider = providerList.filter(providerObj => providerObj['provider-id'] === providerId);
     if (isProvider.length === 0) throw new Error(`Provider [${providerId}] not found`);
 
-    let providerHoldings, id, rootCatalogName;
+    let rootCatalogName = 'CMR-STAC Root catalog';
```

Comment:
is there a reason this value is hard coded in code and not in a configuration or in a presentation layer?

Reply:
In looking at this I don't think the catalog name even needs to specify "CMR-STAC" or "CMR-CLOUDSTAC", all the other link names are generic so I just removed the variable and treated it same as the other link (using "Root catalog" as the name).